lib/drivers/tandemDriver.js

```diff
@@ -1482,7 +1482,7 @@ module.exports = function (config) {
 
       // bolus records
       if (bolus.bolex_size !== undefined || bolus.bolus_option === 'extended') {
-        if (bolus.bolus_size !== undefined || bolus.bolus_insulin_requested !== undefined) {
+        if (bolus.bolus_size !== undefined || bolus.insulin_requested !== undefined) {
```

Comment:
Hmm, this looked reasonable this morning. Now I don't know if I'm second-guessing or onto something. Why is the operator here an `||`? Does `bolus_size` only refer to the `normal` portion of a bolus (so we're looking for either an actual delivered `normal` bolus or a requested one that may have been cancelled before anything was delivered)? That's the only way I think I can make sense out of it.